common/src/main/java/com/wynntils/core/functions/FunctionManager.java

```diff
@@ -247,27 +236,79 @@ private String parseColorCodes(String toProcess) {
         return sb.toString();
     }
 
-    private String doEscapeFormat(String escaped) {
+    private String doEscapeFormat(char escaped) {
         return switch (escaped) {
-            case "\\" -> "\\\\";
-            case "n" -> "\n";
-            case "%" -> "%";
-            case "§" -> "&";
-            case "E" -> EmeraldUnits.EMERALD.getSymbol();
-            case "B" -> EmeraldUnits.EMERALD_BLOCK.getSymbol();
-            case "L" -> EmeraldUnits.LIQUID_EMERALD.getSymbol();
-            case "M" -> "✺";
-            case "H" -> "❤";
+            case '\\' -> "\\\\";
+            case 'n' -> "\n";
+            case '{' -> "\\[\\";
+            case '}' -> "\\]\\";
+            case '§' -> "&";
```

Comment:
Just as a reminder, at least allow `\&` as an escape for `&`, and possibly remove this odd § one.

Also, better fallback than null for unknown escape characters.